laurent22/joplin
# Chore: Mobile: Add note screen tests

# Summary

This is a follow-up pull request to https://github.com/laurent22/joplin/pull/10747. It adds basic `Note` screen tests, including:
- Changing the title of a note updates the title in the database.
- Pressing "delete" moves a note to the trash.
- "Delete" is disabled in read-only notes.

# Manual testing

On Android 13:
1. Start Joplin.
2. Verify that a markdown note renders.
3. Verify that a small edit to a markdown note is saved.
4. Open the note editor.
5. Enable voice typing.
6. Verify that voice typing adds spoken text to the current note.
